<launch>
	<remap from="odom" to="RosAria/pose" />
	<param name="laserTopic" value="base_scan" />
	<param name="ariaFactor" value="1" />
	<param name="simulation" value="True" />
	<param name="amcl/initial_pose_x" value="-2.0" />
	<param name="amcl/initial_pose_y" value="-0.5" />
	<param name="amcl/initial_pose_a" value="0" />
	
	<!-- start gazebo -->
	<include file="$(find pioneer_hallway)/launch/gazebo_small_world.launch"/> 
	 
	<!-- controller node -->
	<include file="$(find pioneer_ros)/launch/pioneer_controller.launch"/>
	
	<!-- start amcl node -->
	<include file="$(find pioneer_hallway)/launch/amcl_pioneer.launch"/>
	
	<!-- start map server -->
   <arg name="map_file" default="$(find pioneer_hallway)/maps/gazebo_sim.yaml"/>
   <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />
	
	<!-- start rviz -->
	<include file="$(find pioneer_hallway)/launch/rviz_pioneer_hallway.launch">
		<arg name="rviz_file" value="gazebo_pioneer_hallway.rviz" />
	</include>
</launch>
